Dockerfile

```diff
@@ -18,6 +18,7 @@ ONBUILD ARG DEPTH_MERGE=100
 ONBUILD ARG CLEAN=true
 ONBUILD ARG COMPILE=true
 ONBUILD ARG CONFIG_BUILD=true
+ONBUILD ARG AUTO_REQUIREMENTS=true
```

Comment:
The issues are during build too, which is the perfect time to resolve them. It really isn't a problem when you're setting up the box, only when a bunch of pre-setup boxes get the wrong default IMO